configs/jsactions/rollup.config.js

```diff
@@ -117,14 +117,28 @@ export default async args => {
     }
 
     async function copyJsModule(from, to) {
-        await copy(from, to, {
-            filter: async path =>
-                (await promises.lstat(path)).isDirectory()
-                    ? !/^(android|ios|windows|macos|\.?(github|gradle)|__(tests|mocks)__|docs|jest|examples?)$/.test(
-                          basename(path)
-                      )
-                    : /.*\.(jsx?|json|tsx?)$/.test(extname(path)) || basename(path).toLowerCase().includes("license")
-        });
+        return new Promise((resolve, reject) =>
+            copy(
+                from,
+                to,
+                {
+                    filter: [
+                        "**/*.{js,jsx,ts,tsx,json}",
+                        "license",
```

Comment:
sometimes there is a `.txt` or `.md` extension. Also maybe nice to align style with the js/json - either use `**` or not. You also can collapse upper and lower case `license` into one option - maybe is nicer for readability

Reply:
Fixed